stable/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: prometheus
-version: 10.4.0
-appVersion: 2.15.2
+version: 10.4.1
```

Comment:
I think this should be major version. 10.5.0.

Reply:
Sounds good, I'll actually open a new PR, something weird about this one and keeps failing the e2e test. 🤷‍♂